### DISCUSSION TITLE
Command "filament:optimize" is not defined

### DISCUSSION THREAD

**Discussion by MRivaldiAP** [filamentphp/filament#14283] [Help] [answered]
### Package

Other

### Package Version

v3.2

### How can we help you?

I have issue when i try to run `php artisan filament:optimize` it always returning with 

> Command "filament:optimize" is not defined. Did you mean one of these?

when i try to run `php artisan filament` it also just returning 4 available function `filament:assets`, `filament:check-translation`, `filament:install`, `filament:upgrade`

i've tried running `php artisan filament:upgrade` and `composer update` but issue still persist

**Comment by leandrocfe** [comment]
Run `composer update -W` and try again.

If it isn't working, run `php artisan about --only=Filament`  .Post the output

**Reply by MRivaldiAP** [reply]
its still not working, here is what i get when running filament about

  Filament ...............................................................................................................................  
  Packages ............................................................................... filament, forms, notifications, support, tables  
  Version ......................................................................................................................... v3.2.0  
  Views .................................................................................................................... NOT PUBLISHED

**Reply by leandrocfe** [reply, marked as answer]
check your `composer.json` file. Make sure you have `^` in the Filament version:

```json
"filament/filament": "^3.2"
```

Run `composer update -W` again
